commands/inventory/inventory.go

```diff
@@ -29,4 +29,38 @@ var (
 	inventoryListC   = inventoryC.Command("list", "List inventory commands").Hide(hide)
 	inventoryAddC    = inventoryC.Command("add", "Add to inventory commands").Hide(hide)
 	inventoryRemoveC = inventoryC.Command("remove", "Remove from inventory commands").Hide(hide)
+	// Add command gobal flags
+	AddAddressFlag     = inventoryAddC.Flag("host", "Hostname").String()
+	AddPortFlag        = inventoryAddC.Flag("port", "Port number").Int64()
+	AddServiceNameFlag = inventoryAddC.Flag("service-name", "Service name").String()
 )
+
+type getter interface {
+	GetAddress() string
+	GetServiceName() string
+	GetPort() int64
+}
+
+// Types implementing the getter interface:
+// - addServiceMySQLCommand
+// - addServiceProxySQLCommand
+// - addServiceMongoDBCommand
+// - ddServicePostgreSQLCommand
+func processGlobalAddFlags(cmd getter) (string, string, int64) {
```

Comment:
@AlekSi, @rnovikovP  do we need this feature for inventory commands?

Reply:
It is not _required_ for this ticket. But I would prefer to move away from positional arguments in most places – practice showed that they are too confusing.
So that depends on how complicated implementation is.

Reply:
In the end, I will revert changes in inventory commands – this PR implements them only for Services, leaving Nodes and Agents with only positional arguments. We need to think more about them.

Reply:
I made it this way because of this comment:
https://github.com/percona/pmm-admin/pull/86#discussion_r358405095

Reply:
Both are correct. We don't want ports for Nodes, but we also don't want replacements for positional arguments only for some commands, and not for others.